.changesets/docs_eh_docs_router_renaming_conventions.md

```diff
@@ -0,0 +1,8 @@
+### Update router naming conventions ([PR #5400](https://github.com/apollographql/router/pull/5400))
+
+Renames our router product to distinguish between our OSS and commercial offerings. Instead of referring to the **Apollo Router**, we now refer to either one of these two:
+- **Apollo Router Core** is Apollo’s free-and-open implementation of a routing runtime for supergraphs. 
+- **GraphOS Router** is based on the Apollo Router Core but fully integrated with GraphOS. GraphOS Routers provide access to GraphOS’s commercial runtime features.
```

Comment:
```suggestion
### Update router naming conventions ([PR #5400](https://github.com/apollographql/router/pull/5400))

Renames our router product to distinguish between our non-commercial and commercial offerings. Instead of referring to the **Apollo Router**, we now refer to the following:
- **Apollo Router Core** is Apollo’s free-and-open (ELv2 licensed) implementation of a routing runtime for supergraphs. 
- **GraphOS Router** is based on the Apollo Router Core and fully integrated with GraphOS. GraphOS Routers provide access to GraphOS’s commercial runtime features.
```